docs/source/requirements.txt

```diff
@@ -0,0 +1 @@
+Sphinx>=1.4.3
```

Comment:
this will likely fail the "rat" Apache license test.   Does this file support a comment block?   If so,  please add one similar to the conf.py header.   If not,  add to the .rat-excludes file at the top level

Reply:
Added.